PalasoUIWindowsForms.Tests/ClearShare/CreativeCommonsLicenseTests.cs

```diff
@@ -112,6 +112,22 @@ public void HasChanges_CanToggle()
 			l.HasChanges = true;
 			Assert.IsTrue(l.HasChanges);
 		}
+		[Test]
+		public void GetDescription_NoTranslation_GivesEnglish()
+		{
+			var l = new CreativeCommonsLicense(true, true, CreativeCommonsLicense.DerivativeRules.DerivativesWithShareAndShareAlike);
+			Assert.AreEqual(l.GetDescription("en"), l.GetDescription("qx"));
 
+		}
+		[Test]
+		public void GetDescription_DontHaveFirstChoice_GivesEnglish()
+		{
+			var l = new CreativeCommonsLicense(true, true, CreativeCommonsLicense.DerivativeRules.DerivativesWithShareAndShareAlike);
+			string usedLanguage;
+			//Assert.AreEqual(l.GetDescription("en"), l.GetDescription(new string[] { "qx", "en" }, out usedLanguage));
+			l.GetDescription(new string[] {"qx", "en"}, out usedLanguage);
+			Assert.AreEqual("en", usedLanguage);
+
+		}
```

Comment:
Also perhaps it would be useful to have a test demonstrating (if it is intentional) that you'd better include English in the list, otherwise you may well get the "missing ID" result.